demo/index.html

```diff
@@ -2,12 +2,20 @@
 <html>
 
 <head>
+  <link href="https://fonts.googleapis.com/css?family=Material+Icons&display=block" rel="stylesheet">
   <script src="https://unpkg.com/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
   <script type="module" src="../lib/code-sample-editor.js"></script>
+  <style>
+    code-sample-editor {
+      margin-bottom: 48px;
+    }
+  </style>
 </head>
+
 <body>
-  <code-sample-editor project-path="./templates"></code-sample-editor>
-  <code-sample-editor project-path="./templates"></code-sample-editor>
-  <code-sample-editor project-path="./templates" sandbox-scope="myScope"></code-sample-editor>
+  <!-- <code-sample-editor project-src="./typescript/project.json"></code-sample-editor> -->
+  <code-sample-editor project-src="./project-1/project.json"></code-sample-editor>
+  <code-sample-editor project-src="./mwc-button/project.json"></code-sample-editor>
```

Comment:
I changed the attribute name to end in `src` so it has similar semantics to built-in `src` attributes, and the value to be a file so we don't have a filename we look for. From the config, demo files just follow relative paths.